src/ertai/creature.py

```diff
@@ -0,0 +1,96 @@
+"""
+Class for Creature
+"""
+
+from .card import Card
+from .mana import Mana
+
+
+class Creature(Card):
+    """
+    A class for basic creatures.
+    """
+
+    def __init__(
+        self,
+        title=None,
+        cost=Mana(),
+        tapped=False,
+        power=0,
+        toughness=0,
+    ):
+        """
+        Initiate this creature by giving power, toughness of it.
+
+        Other basic information below of this Creature derived from Card:
+
+        title : title of this creature
+
+        cost : cost to cast this creature
+
+        tapped : tap information of this creature
+
+        Parameters:
+            - power : power  of this creature
+            - toughness : Toughness value of this creature,
+                but this value can be modified in a fight or other condition
+            - base_toughness : Toughness value of this creature,
+                use to store the base toughness of this creature
+        """
+
+        Card.__init__(self, title, cost, tapped)
```

Comment:
Instead of using a base Python class can we use the `@dataclass` approach which is how the Land card was done: https://github.com/drvinceknight/ertai/blob/main/src/ertai/card.py#L44